modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -234,6 +234,31 @@ kubectl create secret docker-registry rtf-pull-secret --namespace <rtf-namespace
 . Follow the *Helm* path instructions to download the `values.yml` file so that you can modify its values. For the private registry configuration, update `rtfRegistry` with your local docker server, and update the `pullSecretName` parameter. . 
 . Continue with the *Helm* path instructions and install Runtime Fabric in your Kubernetes cluster.
 
+--
+
+OpenShift::
++
+--
+. <<synch-rtf,Synchronize your container images>> to your local registry.
+. xref:install-helm.adoc#create-a-runtime-fabric-using-runtime-manager[Create a Runtime Fabric using Runtime Manager].
+. In Runtime Manager, select the *Helm* path, and follow the instructions on the screen platform.
```

Comment:
There is no helm path to be selected here.

We should mention to follow the installation steps shown in runtime manager.